kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -22,6 +47,7 @@ export class JavascriptErrorReport extends ErrorReport {
     return {
       error_message: this.e.error.message,
       traceback: this.e.error.stack,
+      context: this.getContext(),
```

Comment:
Looks like we have repeated implementations here and for the next class, any reason this is doesn't just live in the parent class?